packages/desktop/components/popups/AddNodePopup.svelte

```diff
@@ -30,6 +33,13 @@
             } else {
                 await addNodeToClientOptions(node)
             }
+
+            if (Platform.isFeatureFlagEnabled('governance')) {
+                if (currentNode?.url !== node?.url) {
```

Comment:
Why do we check if its the current node, if a node has been added or edited then we should pull in the events